DemoApp/appcenter-post-clone.sh

```diff
@@ -3,3 +3,6 @@ echo "Executing post clone script in `pwd`"
 echo $NPM_RC | base64 --decode > $APPCENTER_SOURCE_DIRECTORY/DemoApp/.npmrc
 # Delete everything except DemoApp folder
 rm -rf ../appcenter* ../AppCenterReactNativeShared ../TestApp34 ../BrownfieldTestApp ../TestApp
+
+pod repo add $REPO_NAME https://$USER_ACCOUNT:$ACCESS_TOKEN@msmobilecenter.visualstudio.com/SDK/_git/$REPO_NAME
```

Comment:
not sure why repo name needs to be a variable since msmobilecenter.visualstudio.com is harcoded.
Either hardcode both or variables for both.

Reply:
It could be anything but we are reusing the repo name as its name on VM since it shouldn't conflict.